CMakeLists.txt

```
@@ -101,9 +103,15 @@ if(CYGWIN)
     NAMES cspice.a csupport.a
     PATHS ${CSPICE_DIR}/cspice_cygwin/lib)
 elseif(UNIX)
-  find_library(CSPICE_LIB
-    NAMES cspice.a csupport.a
-    PATHS ${CSPICE_DIR}/cspice_unix/lib)
+  if(BUILD_64BIT)
+    find_library(CSPICE_LIB
+      NAMES cspice.a csupport.a
+      PATHS ${CSPICE_DIR}/cspice_unix_64bit/lib)
```

[IMO]
32,64bitの区別，こんなかんじにやりがちです(これで十分伝わるし短くて済むので)
```suggestion
      PATHS ${CSPICE_DIR}/cspice_unix64/lib)
```

`lib_64bit`とかも`lib64`でいいと思います